xbmc/interfaces/python/PythonInvoker.cpp

```diff
@@ -147,7 +147,12 @@ bool CPythonInvoker::execute(const std::string &script, const std::vector<std::s
 
   // copy the arguments into a local buffer
   m_argc = arguments.size();
-  m_argv = new char*[m_argc];
+
+  if(m_argc > 0)
+  {
+    m_argv = new char*[m_argc];
```

Comment:
I figured this wasn't a regression so I didn't change it. I don't mind making the change though. Will send another commit in a little while.